RecoTauTag/HLTProducers/src/L1PFJetsMatching.cc

```diff
@@ -0,0 +1,148 @@
+#include "RecoTauTag/HLTProducers/interface/L1PFJetsMatching.h"
+#include "Math/GenVector/VectorUtil.h"
+#include "DataFormats/HLTReco/interface/TriggerTypeDefs.h"
+#include "FWCore/Utilities/interface/EDMException.h"
+#include "DataFormats/JetReco/interface/PFJet.h"
+
+//
+// class decleration
+//
+using namespace reco   ;
+using namespace std    ;
+using namespace edm    ;
+using namespace trigger;
+
+std::pair<PFJetCollection,PFJetCollection> Categorise(PFJetCollection CaloL2jets, double pt1, double pt2, double Mjj)
+{
+
+int A=0;
+    std::pair<PFJetCollection,PFJetCollection> output;
+   unsigned int i1 = 0;
+   unsigned int i2 = 0;
+    double mjj = -9999;
+    if (CaloL2jets.size()>1){
+    for (unsigned int i = 0; i < CaloL2jets.size()-1; i++)
+        for (unsigned int j = i+1; j < CaloL2jets.size(); j++)
+    {
+        const PFJet &  myJet1 = (CaloL2jets)[i];
+        const PFJet &  myJet2 = (CaloL2jets)[j];
+        
```

Comment:
while we're at it, CaloL2jets is not a good name for a PFJetCollection 
(although I know why it ended up being named this way)